modules/ephemeral-storage-csi-inline-overview-admin-plugin.adoc

```diff
@@ -0,0 +1,106 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/ephemeral-storage-csi-inline.adoc
+
+:_content-type: CONCEPT
+[id="ephemeral-storage-csi-overview-admin-plugin_{context}"]
+= CSI Volume Admission plug-in
+
+The CSI Volume Admission plug-in provides a mechanism where the use of an individual CSI driver capable of provisioning CSI ephemeral volumes can be restricted on pod admission. Administrators or distributions can optionally add a `csi-ephemeral-volume-profile` label to a `CSIDriver` object, and the label is then inspected by the Admission plug-in and used in enforcement, warning, and audit decisions.
+
+== Overview
```

Comment:
Headings need an ID or a `[discrete]` tag per [OpenShift doc guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#assemblymodule-titles-and-section-headings). Maybe also include what it is an overview of.